src/App.tsx

```diff
@@ -1,12 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { getAll } from './api/goods';
+import { Good } from './react-app-env';
+import { GoodsList } from './components/GoodsList';
+
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+const App: React.FC = () => {
+  const [allGoods, setAllGoods] = useState<Good[]>([]);
+
+  function getAllGoods() {
+    getAll()
+      .then((goods) => setAllGoods(goods));
+  }
+
+  function get5First() {
```

Comment:
I think, that this function (with sorting and slicing) should be in goods.ts file, and here you should only change the state. But I'm not sure if it's a mistake if that logic is in the component App.